src/microbe_stage/Microbe.cs

```diff
@@ -639,14 +639,16 @@ public void EmitToxin(Compound agentType = null)
 
         float amountAvailable = Compounds.GetCompoundAmount(agentType);
 
-        if (amountAvailable < Constants.MINIMUM_AGENT_EMISSION_AMOUNT)
+        // Emit as much as you have, but don't start the cooldown if that's zero
+        float amountEmitted = Math.Min(amountAvailable / Constants.MAXIMUM_AGENT_EMISSION_AMOUNT, 1.0f);
```

Comment:
Why is the divide by the constant here?
Wouldn't it be simpler to just get the min from the available amount and the maximum amount to emit.
Then in the if just return if the amount is less than the min emission amount (see my last comment about that).

Reply:
If you really want to keep this code, then "amountEmitted" *must* be renamed as it is not actually the amount, but a fraction of the maximum emit amount.

Reply:
Looking at it a second time, I agree that it was very strange. It made sense to me at the time.